### PR TITLE
implement Headers section

### PR DESCRIPTION
**Pull request by farhan-sauce**
**Feature**: implement Headers section in request detail

Screen Shot 2020-01-06 at 10 22 26 PM

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -56,6 +56,18 @@ export const getContent = ({ mimeType, text }) => {
...
+export const sortHeaders = (current, next) => {
+  if (current.name < next.name) {
+    return -1;
+  }
+  return current.name > next.name ? 1 : 0;
+};
+
+export const getHeaders = (entry) => ({
+  request: entry.request.headers.sort(sortHeaders),
+  response: entry.response.headers.sort(sortHeaders),
```
Comment (seanenright) [inline src/utils.js]: tests? 
also aren't headers.name a string? why do we need to sort them?

File: src/Containers/ReqDetailContainer.styles.scss [hunk 2]
```diff
@@ -20,3 +21,11 @@ $close-button-width: 35px;
   top: $gutter / 2;
   cursor: pointer;
 }
+
+.tabs-container {
+  display: flex;
+  width: 100%;
+  padding: $gutter;
```
Comment (seanenright) [inline src/Containers/ReqDetailContainer.styles.scss]: $gutter - maybe we can just fix this later

File: src/utils.js (hunk 1 above)
Comment (farhan-sauce) [reply src/utils.js]: To be consistent with chrome devtools header section, it's alphabetically sorted.

File: src/Containers/ReqDetailContainer.styles.scss (hunk 2 above)
Comment (farhan-sauce) [reply src/Containers/ReqDetailContainer.styles.scss]: PR to fix this https://github.com/saucelabs/network-viewer/pull/17

**Pull request merged**